packages/es-components/src/components/controls/checkbox/__snapshots__/Checkbox.specs.js.snap

```diff
@@ -2,22 +2,16 @@
 
 exports[`Checkbox renders as expected 1`] = `
 <label
-  className="es-checkbox sc-bwzfXH iaOWyJ sc-bdVaJa lgVfRX"
-  disabled={true}
```

Comment:
It looks like the snapshot test needs to be updated to use `checked` and `disabled`.

Reply:
Updated.  Good catch